src/Greenshot.Editor/Drawing/CropContainer.cs

```diff
@@ -45,13 +70,70 @@ protected override void OnDeserialized(StreamingContext streamingContext)
         }
 
         private void Init()
+        {
+            switch (GetFieldValue(FieldType.CROPMODE))
+            {
+                case CropMode.Horizontal:
+                    {
+                        InitHorizontalCropOutStyle();
+                        break;
+                    }
+                case CropMode.Vertical:
+                    {
+                        InitVerticalCropOutStyle();
+                        break;
+                    }
+                default:
+                    {
+                        InitCropStyle();
+                        break;
+                    }
+            }
+        }
+
+        private void InitCropStyle()
         {
             CreateDefaultAdorners();
         }
 
+        private void InitHorizontalCropOutStyle()
+        {
+            var defaultHeight = 25;
```

Comment:
I'd prefer "const" here, as that is what it is... I do wonder if there is a need to make such values scalable like depending on the DPI.